qa/L0_identity/test.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash -ex
 
-echo "Test run"
-python identity_client.py
+GRPC_ADDR:=${1:-"localhost:8001"}
```

Comment:
Is it also intended that it can be specified as first argument? 
Otherwise I would expect just:
```suggestion
GRPC_ADDR:=${GRPC_ADDR:-"localhost:8001"}
```
or something like that. Not that it's very important.

Reply:
Yup, you may use it in 2 ways
```bash
GRPC_ADDR='localhost:8001' bash test.sh
bash test.sh 'localhost:8001'
```